docs/libraries/azure-core/reference/data-types.md

```diff
@@ -231,6 +239,30 @@ Provides the most common query parameters for list operations.
 model Azure.Core.StandardListQueryParameters
 ```
 
+### `StatusMonitorOptions` {#Azure.Core.StatusMonitorOptions}
+
+Options for Lro status monitors.
+
+```typespec
+model Azure.Core.StatusMonitorOptions
+```
+
+### `StatusMonitorPollingOptions` {#Azure.Core.StatusMonitorPollingOptions}
+
+Options for overriding a polling endpoint that uses a StatusMonitor
+
+```typespec
+model Azure.Core.StatusMonitorPollingOptions<TPollingModel, TFinalResult, TFinalProperty>
```

Comment:
Didn't Will just merge his PR that changes the settled on not using the T prefix for template args?

Reply:
Yeah, and since this is a new template, it should use the new paradigm, good catch